src/server/api/routers/chat.ts

```diff
@@ -43,7 +43,7 @@ function getListHandler() {
 
       const chats = await db.query.chats.findMany({
         where: eq(schema.chats.threadId, input.threadId),
-        orderBy: [desc(schema.chats.id)],
```

Comment:
Why we need to change loading chat from oldest to newest ?

Reply:
We make confuse at here, table will render newest to oldest thread chat. => Then i replace "asc" to "desc" at api/routers/thread.ts files.  
DetailDrawer will show oldest to newest chat like chat history by using "asc" at api/routers/chat.ts. 
![image](https://github.com/dwarvesf/chatbot-builder/assets/73730153/85e6890a-9ded-4384-84a5-e622bce33e9d)